### ISSUE TITLE
Repeat debug calls to geth

### ISSUE THREAD

**Issue by Vid201** [Vid201/aa-bundler#75]
`debug_traceCall` sometimes returns empty response - in that case, request should be repeated after some timeout.

**Comment by zsluedem** [comment]
@Vid201 Just curious. I think this is a good one. Is it fixed in your work now?

**Comment by Vid201** [comment]
> @Vid201 Just curious. I think this is a good one. Is it fixed in your work now?

No. But I thought it's happening very often (at least once at 10 calls), but it turned out something else was wrong. I'm not sure now how often that happens. We can leave it open to add something in the future.

**Comment by brohamgoham** [comment]
@Vid201 I see you reopend. Can i take this issue or is it taken?.

**Comment by Vid201** [comment]
> @Vid201 I see you reopend. Can i take this issue or is it taken?.

Hi @brohamgoham . Feel free to take it, thanks for planning to contribute to the project. If you will any questions, make sure to join the TG group: https://t.me/aabundler

**Comment by brohamgoham** [comment]
@Vid201 

> > @Vid201 I see you reopend. Can i take this issue or is it taken?.
> 
> Hi @brohamgoham . Feel free to take it, thanks for planning to contribute to the project. If you will any questions, make sure to join the TG group: https://t.me/aabundler

Hey i did join telegram but it is subscribe only, I cannot message in there

**Comment by Vid201** [comment]
> @Vid201
> 
> > > @Vid201 I see you reopend. Can i take this issue or is it taken?.
> > 
> > 
> > Hi @brohamgoham . Feel free to take it, thanks for planning to contribute to the project. If you will any questions, make sure to join the TG group: https://t.me/aabundler
> 
> Hey i did join telegram but it is subscribe only, I cannot message in there

Yeah, my bad. Thanks for letting me know
